learn/app-development/widgets/datalive/live-form/liveform-layouts.md

```diff
@@ -43,5 +45,9 @@ For form fields, the **alignment of the caption** can be configured to place as
 ### 3. Column Form with top-aligned Captions 
     
 <iframe width="100%" height="450" style="background-color: snow;" allowtransparency="true" src="https://apps.wavemakeronline.com/documentation_snippets/#/LiveFormwithThreeCol">3-column Form</iframe>
-    
+
+### 4. Column Form with floating Captions 
```

Comment:
Lets add in brackets (Angular material style)

Reply:
Added